tests/testthat/test-data_types.R

```diff
@@ -136,7 +136,7 @@ test_that("all data types work", {
     type_array_bigint=NA,
     type_map_varchar_bigint=NA
   )
-  attr(e[['type_timestamp_with_timezone']], 'tzone') <- NA_character_
+  attr(e[['type_timestamp_with_timezone']], 'tzone') <- NULL
```

Comment:
See the [commit message](https://github.com/prestodb/RPresto/pull/48/commits/301ce60b653a99617d3e84cd67c2da84821c36bd) for the corresponding commit.

Two reasons:
- R 3.1.x complains with NA timezone.
- Let's say you have a 0-row with US/Pacific and 1-row with US/Eastern. dplyr::bind_rows() switches to UTC when they differ.